odoo/models.py

```diff
@@ -4363,7 +4377,7 @@ def _filter_access_rules_python(self, operation):
         dom = self.env['ir.rule']._compute_domain(self._name, operation)
         return self.sudo().filtered_domain(dom or [])
 
-    def unlink(self):
+    def unlink(self) -> typing.Literal[True]:
```

Comment:
I wonder whether this one is a good idea, because
- the returned value is irrelevant (nobody checks the returned value);
- it is not guaranteed to be `True` (overrides);
- ideally it should be `None`.

Reply:
Indeed, we don't need to type this now and when xmlrpc is decommissioned, we can return `None`.